torch/csrc/cuda/Module.cpp

```diff
@@ -873,6 +873,12 @@ PyObject* THCPModule_cudaGetSyncDebugMode(PyObject* self, PyObject* noargs) {
   END_HANDLE_TH_ERRORS
 }
 
+static PyObject* THCPModule_hasMagma(PyObject* self, PyObject* noargs) {
+  HANDLE_TH_ERRORS
+  return at::hasMAGMA() ? Py_True : Py_False;
```

Comment:
This is missing refcount bumps!
Py_RETURN_TRUE and cie needs to be used

Reply:
I've moved the code to `Module.cpp`, so it still returns `Py_True` but it should not matter, should it?